.github/workflows/release.yml

```diff
@@ -0,0 +1,62 @@
+name: "publish"
```

Comment:
I don't like these workflows. First these are broken. Second dev is missing which is essential at this stage of development.
Let's remove these and add simple ones based on:
https://github.com/mintlayer/mintlayer-core/blob/master/.github/workflows/build.yml
and
https://github.com/mintlayer/mintlayer-core/blob/master/.github/workflows/code_checks.yml